### DISCUSSION TITLE
code server, config.yaml do not exist when start with systemctl

### DISCUSSION THREAD

**Discussion by FranciscoOssian** [coder/code-server#5748] [Q&A]
Hi, I was trying to setup Code-server with Ngix.

```
#!/bin/bash

curl -fsSL https://code-server.dev/install.sh | sh

sudo systemctl enable --now code-server@$USER

sudo apt update
sudo apt install -y nginx certbot python3-certbot-nginx

echo your domain:
read domain
echo your email:
read email

echo "
server {
    listen 80;
    listen [::]:80;
    server_name $domain;

    location / {
      proxy_pass http://localhost:8080/;
      proxy_set_header Host \$host;
      proxy_set_header Upgrade \$http_upgrade;
      proxy_set_header Connection upgrade;
      proxy_set_header Accept-Encoding gzip;
    }
}" > /etc/nginx/sites-available/code-server

sudo ln -s ../sites-available/code-server /etc/nginx/sites-enabled/code-server
sudo certbot --non-interactive --redirect --agree-tos --nginx -d $domain -m $email

```

All starts fine, but `~/.config/code-server/config.yaml` do not exist (even with Code-server after First running)

`sudo systemctl enable --now code-server@$USER`

If I try Just run
`code-server`, the file is created but the password don't match.

How can I fix this, and create the config file when execute my .sh?

Ngix works, code-server starts without config file yaml (the file with password)

**Comment by jsjoeio** [comment]
> How can I fix this, and create the config file when execute my .sh?

The naive me would try something like:
```shell
mv default.yaml ~/.config/code-server/config.yaml
```

where `default.yaml` has your starter config but guessing you tried that?
